src/app/api/auth/mobile/logout/route.ts

```diff
@@ -0,0 +1,28 @@
+import { NextRequest, NextResponse } from 'next/server';
+import db from '@/db';
+
+export async function POST(request: NextRequest) {
+  try {
+    const body = await request.json();
```

Comment:
i was testing this, why do i have to pass userId here? 
i saw that in https://github.com/code100x/cms/pull/772/files#diff-85f5526f057dabf2a9a7e00ec553a8049306f43a9f343ff532fb5657121c7791

u are setting userId in search params, can't we use that?

Reply:
Oh yes, we can use that, I added the url params later, and forgot to change the logout route, could you please change it? Or should I do it?